samples/deployment/test-network/Makefile

```diff
@@ -7,8 +7,8 @@ TOP = ../../..
 include $(TOP)/build.mk
 
 
-TEST_CC_ID ?= echo
-TEST_CC_PATH ?= ${FPC_PATH}/samples/chaincode/${TEST_CC_ID}
+CC_ID ?= echo
+CC_PATH ?= ${FPC_PATH}/samples/chaincode/${CC_ID}
```

Comment:
It would be nice to check if `CC_ID` and `CC_PATH` are set; otherwise abort make, similar to what we are doing in `installFPC.sh`.

Reply:
Done!

Reply:
@mbrandenburger I ended up doing it a bit differently than in other parts but it works. Please can you review.